### PR TITLE
google login now working on backend

### PR DESCRIPTION
**Pull request by phemmylintry**

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -177,7 +177,7 @@
 MEDIA_URL = '/media/'
 MEDIA_ROOT = str(APPS_DIR('media'))
 
-SITE_ID = 1
+SITE_ID = 2
```
Comment (KeelTech) [inline config/settings/base.py:180]: 2?

File: config/settings/base.py [hunk 2]
```diff
@@ -234,7 +235,7 @@
 }
 
 JWT_AUTH = {
-    'JWT_AUTH_HEADER_PREFIX': 'JWT',
+    'JWT_AUTH_HEADER_PREFIX': 'bearer',
```
Comment (KeelTech) [inline config/settings/base.py:238]: great!!

File: keel/authentication/backends.py [hunk 3]
```diff
@@ -57,9 +57,9 @@ def authenticate(self, request):
 
 
     def _authenticate_credentials(self, request, token):
-        user_key = settings.USER_SECRET_KEY
+        # user_key = "settings.USER_SECRET_KEY" # No USER_SECRET_KEY in settings
```
Comment (KeelTech) [inline keel/authentication/backends.py:60]: then you can add it  :)  ..in base.py

File: config/settings/base.py (hunk 1 above)
Comment (phemmylintry) [reply config/settings/base.py:180]: I got an error when it was on 1

File: keel/authentication/models.py [hunk 4]
```diff
@@ -43,7 +43,7 @@ def create_superuser(self, email, password, **extra_fields):
 
 
 class User(AbstractBaseUser, PermissionsMixin):
-    username = None
+    username = models.CharField(max_length=255, blank=False, null=True, default=None)
```
Comment (KeelTech) [inline keel/authentication/models.py:46]: do we need the user_name field again?

**Review by KeelTech — APPROVED** [review]

**Pull request merged**